load_organisms_and_genes_to_db.sh

```diff
@@ -0,0 +1,244 @@
+# Create organism records
+python manage.py organisms_create_or_update --taxonomy_id=9606 \
+    --scientific_name="Homo sapiens" --common_name="Human"
+python manage.py organisms_create_or_update --taxonomy_id=4932 \
+    --scientific_name="Saccharomyces cerevisiae" --common_name="S. cerevisiae"
+python manage.py organisms_create_or_update --taxonomy_id=10090 \
+    --scientific_name="Mus musculus" --common_name="Mouse"
+python manage.py organisms_create_or_update --taxonomy_id=10116 \
+    --scientific_name="Rattus norvegicus" --common_name="Rat"
+python manage.py organisms_create_or_update --taxonomy_id=6239 \
+    --scientific_name="Caenorhabditis elegans" --common_name="C. elegans"
+python manage.py organisms_create_or_update --taxonomy_id=3702 \
+    --scientific_name="Arabidopsis thaliana" --common_name="Arabidopsis"
+python manage.py organisms_create_or_update --taxonomy_id=7227 \
+    --scientific_name="Drosophila melanogaster" --common_name="Fruit fly"
+python manage.py organisms_create_or_update --taxonomy_id=7955 \
+    --scientific_name="Danio rerio" --common_name="Zebrafish"
+python manage.py organisms_create_or_update --taxonomy_id=208964 \
+    --scientific_name="Pseudomonas aeruginosa" --common_name="Pseudomonas aeruginosa"
+
+#CREATE XRDBS
+python manage.py genes_add_xrdb --name=Ensembl \
```

Comment:
Same question: What about a "for" loop?

Reply:
Actually for this one, would it really make sense to create a for loop? According to the link you posted, I would need to declare an associative array for each of the xrdbs, which would take as many lines as just calling these commands, no?